### ISSUE TITLE
`pharmpy.modeling.calculate_eps_shrinkage`

### ISSUE THREAD

**Issue by king-of-poppk** [pharmpy/pharmpy#3389]
### Feature request

Is there an equivalent to `pharmpy.modeling.calculate_eta_shrinkage` for epsilons?

**Comment by rikardn** [comment]
No currently not. It would be calculated as `1 - stdev(IWRES)`. Perhaps it would be best to parse it from the `lst` file.

**Comment by king-of-poppk** [comment]
There is also an optional `.shk` file if I am not mistaken. Should shrinkages in general be abstracted away from their source?

**Comment by rikardn** [comment]
Yes it is probably easier to parse from the `.shk` file. 

If parsed from the estimation tool output we would need to add it to `ModelfitResults`.

**Comment by king-of-poppk** [comment]
Is shrinkage natural to attach to parameter estimates? Or is it more something that belongs to their corresponding random variables?

**Comment by rikardn** [comment]
Yes it is connected to the distribution of the random variables.

**Comment by king-of-poppk** [comment]
OK. So should this be stored at `results.shrinkage`, keyed by items from `model.random_variables`?

**Comment by rikardn** [comment]
Yes, something like that. Shrinkage, at least for etas, can be presented on multiple (2?) scales. I don't know if we need to have all scales or if one makes more sense than the others.
